### PR TITLE
Dev bt 2053 - pin jquery and angular versions

### PR DESCRIPTION
**Pull request by dmepham**
https://dataxu.atlassian.net/browse/BT-2053

testing in progress

### CODE REVIEW

**Comment by dmepham** [comment]
versions chosen based on the last AMI in which this landing page worked: https://jenkins.devaws.dataxu.net/job/qatools_create_ami/17/console

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CODEOWNERS @dataxu/releng
+Dockerfile @dataxu/releng
+Jenkinsfile @dataxu/releng
```
Comment (dmepham) [inline CODEOWNERS:3]: not sure why this commit was included in my PR, but it should be here anyway so I'll keep it

**Comment by dmepham** [comment]
whoops, meant to put this against dataxus fork. sorry matt! hope all is well

**Comment by ferrants** [comment]
:shipit: lol

**Pull request closed without merging**